services/py-api/src/database/repository/participants_repository.py

```diff
@@ -40,20 +41,34 @@ async def create(
         session: Optional[AsyncIOMotorClientSession] = None,
         **kwargs: Dict[str, Any]
     ) -> Result[Participant, DuplicateEmailError | Exception]:
+
+        participant = Participant(
+            name=input_data.name,
+            email=input_data.email,
+            is_admin=input_data.is_admin,
+            # If the team_id is passed as a kwarg the participant will be inserted in the given team
```

Comment:
This comment should be added as a docstring. This is really important.

Reply:
Fixed in c3390a1c03923cd8ec9f09bcc4f47a9a0ac2942d

Reply:
It is added to the base "create" method docs:
<img width="898" alt="image" src="https://github.com/user-attachments/assets/fd1b222a-ba5f-414d-9cee-9c65273ca38f" />